added documentation (#97)
parameterIT/tool: documentation/docs/introduction.md

## Modu

This tool provides the possibility for developers and/or businesses to configure a specific software quality model directly into a flexible and modular Software Quality Assessment Tool.

The Modu tool allows for quick implementation of new metrics and is fully customizable from the low-level metrics measured in the actual source code to the high-level aggregation formulas that depict the overall _quality_ of the source code.

> The tool has been developed as an open source project under the AGPL-3.0 license, and welcomes all additions made by external contributers. 

Reviewer: ```suggestion
The tool is open source project under the AGPL-3.0 license, and welcomes all additions made by external contributers. 
```
